src/app/locationForm/number/LocationNumber.jsx

```diff
@@ -42,7 +42,7 @@ class LocationNumberEdit extends Component {
   }
 
   onSubmit() {
-    this.props.updateValue({
+    this.props.updateValue(this.props.value.id, {
```

Comment:
If the location doesn't already have a phone, won't this produce a `Cannot read property 'id' of undefined` error?

Reply:
Fixed in 3ffaa7cfd008343d6c966f10d3416ef4d943f154. Thanks.

Reply:
Tested on location 04dcd3a8-5ed0-4d2a-a357-56b0703f3c8d, which does not have a phone number set.